Analysis/GeneralTopics/tars-cloud-basic-analysis.md

```diff
@@ -0,0 +1,114 @@
+- GROUP
+	- In order to reduce response time of calls among servers and minimize influence of network failure, tars supports IDC grouping to groups server according to their locations.<br> To facilitate service management, tars groups servers into Sets.
+- CONFIG
+	- Server configurations are managed by tarsweb. Developer can change configurations in a webpage making the change easily and the change is safer. <br> Developer can check history of configuration changes and rollback to previous version.
+- SHIELD
+	- It's implemented with two ways: name service excluded and client shielding. <br>Name service will exclude fail server from address list. And Client removes inactive server from the server list for client shielding.
+- PROTECT
+	- To avoid overloading the system because of burst requests or machine fault, tars handles this scenario in the framework. In order to improve system throughput, server uses request queue to process request asynchronously.
+
+## Supported Platforms :
+- Linux
+- Windows
+- MAC
+
+## Supported Languages :
+- C++
+- Java
+- Nodejs
+- PHP
+- Go
+
+## TARS Projects :
+- TARS LAB
+	- JMeter
+	- Benchmark
+	- TarsJavaStart
+	- TarsTools
+- Service Governance
+	- Tseer
+	- Gateway
+- Storage
+	- DCache
+- Infrastructure
+	- K8Stars
+
+### Link for TARS Landscape: https://tarscloud.org/foundation/projects
+
+# Technical Analysis:
+<img src="https://tarscloud.github.io/TarsDocs_en/assets/tars_top_en.png">
+The above image provides a description of the high-level architecture of TARS cloud.
+1. At the top we have the management system which is an web based interface for managing the platform.<br>
+2. The next layer is the registry which takes care of all the microservices running on the platform and along with their metadata.<br>
+3. The node management module help with the process of deployment, configuration, monitoring and scaling a node on the platform.
+4. TARS cloud also provides support for scalable and dynamic object storage.<br>
+
+### SODA and TARS
+SODA is an open source project which provides an unified framework for managing data storage across different types of storage systems,<br>
+TARS cloud includes built-in support for SODA managed systems, The developers can easily integrate their microservices with SODA-managed<br>
+storage systems using the TARS cloud platform.<br>
+Therefore we can say that SODA and TARS can work seamlessly in building and deploying data-intensive applications that can meet the modern demands<br>
+
+
+
+## Other Links:
+| -- | TARS |
+|--|--|
+| **Website** | https://tarscloud.org/ |
+| **TARS Projects** |  https://tarscloud.org/foundation/projects|
+| **License** | [BSD-3-Clause] Read more below. |
+| **TARS Projects** | https://github.com/TarsCloud |
+| **Twitter** | https://twitter.com/TarsCloud |
+
+
+## License:
```

Comment:
Please mention it is the license of what?
Also, please explain what is the license and why are adding it in the analysis